xml/System.Net.Security/TlsCipherSuite.xml

```diff
@@ -18,8 +18,17 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents cipher suite values for the TLS (formerly SSL) protocol.</summary>
+    <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
+Both field names and values are based on the [TLS Cipher Suites list](https://www.iana.org/assignments/tls-parameters/tls-parameters.xml#tls-parameters-4) from the Internet Assigned Numbers Authority (IANA).
+This enumeration represents values that were known at the time a specific version of .NET was released. Unmapped values may be returned by properties or methods, and methods should generally not verify that a given input was defined within the enum, as the value may be transparently passed from or to the system underlying TLS implementation.
```

Comment:
```suggestion
This enumeration represents values that were known at the time a specific version of .NET was released. Unmapped values may be returned by properties or methods, and methods should generally not verify that a given input was defined within the enum, as the value may be transparently passed from or to the system and the underlying TLS implementation.
```

Reply:
I was trying to refer to the `underlying TLS implementation` within the system.
This might be more appropriate perhaps?

```suggestion
This enumeration represents values that were known at the time a specific version of .NET was released. Unmapped values may be returned by properties or methods, and methods should generally not verify that a given input was defined within the enum, as the value may be transparently passed from or to the system's underlying TLS implementation.
```

Reply:
Yes.